cvat-ui/src/utils/mousetrap-react.tsx

```diff
@@ -34,17 +35,21 @@ export default function GlobalHotKeys(props: Props): JSX.Element {
     const { children, keyMap, handlers } = props;
     useEffect(() => {
         for (const key of Object.keys(keyMap)) {
-            const { sequences, action } = keyMap[key];
-            const handler = handlers[key];
-            Mousetrap.bind(sequences, handler, action);
-            applicationKeyMap[key] = keyMap[key];
+            if (keyMap[key] && keyMap[key].sequences) {
```

Comment:
It seems we do not need this condition anymore, as `subKeyMap` will ignore shortcuts with `undefined` values, right? 

The same in return.